packages/core/src/rest/ZosmfRestClient.ts

```diff
@@ -101,6 +101,10 @@ export class ZosmfRestClient extends RestClient {
                 original.additionalDetails = "Token is not valid or expired.\n\n" +
                     "For CLI usage, see `zowe auth login apiml --help`";
             }
+            // TODO: Add PFX support in the future
+            if (this.session.ISession.type === SessConstants.AUTH_TYPE_CERT_PEM) {
+                original.additionalDetails = "The certificate is not valid or expired.\n\n";
```

Comment:
Sorry for being nitpicky, but should the word `The` be removed to be consistent with the messages for expired password/token?

Reply:
Yes, it probably should be removed.